app/containers/MessageBox/index.js

```diff
@@ -596,6 +642,11 @@ class MessageBox extends Component {
 	}
 
 	showUploadModal = (file) => {
+		const { fullScreenUp } = this.state;
+
+		if (fullScreenUp) {
```

Comment:
If we just have the one value from the state and we are just using it once, then I think it's a bit cleaner to just using `this.state.fullScreenUp` in the `if` statement rather than assigning it to a variable.

Reply:
We have `react/destructuring-assignment` enabled.
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/destructuring-assignment.md